debezium-core/pom.xml

```diff
@@ -101,6 +101,12 @@
             <artifactId>kafka-connect-avro-converter</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- open libraries -->
+        <dependency>
+            <groupId>com.twitter.common</groupId>
+            <artifactId>objectsize</artifactId>
+            <version>0.0.12</version>
```

Comment:
I think a newer version of the ObjectSizeCalculator is available here: https://github.com/twitter/util/blob/develop/util-core/src/test/java/com/twitter/util/ObjectSizeCalculator.java but it's not available on maven central and twitter only uses it for unit tests.